commands/apps/index.js

```diff
@@ -58,18 +56,22 @@ function * run (context, heroku) {
     }
   }
 
-  let requests = yield {
-    apps: org ? heroku.get(`/organizations/${org}/apps`) : heroku.get('/apps'),
-    user: heroku.get('/account')
+  let path = '/users/~/apps'
+  if (org) path = `/organizations/${org}/apps`
+  else if (space || context.flags.all) path = '/apps'
```

Comment:
@dickeyxxx I am pretty sure this will never execute if space is true due to the fact that org has a value if space has a value